### ISSUE TITLE
[Feature Request] File transfer thumbnail

### ISSUE THREAD

**Issue by NicoHood** [qTox/qTox#2794]
A friend suggested to add a thumbnail to pictures that you send over tox. This would be really useful. (used linux)

**Comment by zetok** [comment]
![thumbnail](https://cloud.githubusercontent.com/assets/3148759/12311703/ffb06d04-ba50-11e5-81bb-eee51fe2018d.png)

Do you have on mind something other/more?

**Comment by TheNain38** [comment]
I think, he meant a tumbnail that is displayed before actually downloading the picture (e.g. like on skype)...

**Comment by ProMcTagonist** [comment]
That would involve autoaccepting a thumbnail file transfer and sounds like it may be in the purview of [core](https://github.com/irungentoo/toxcore/issues).

**Comment by NicoHood** [comment]
Yes, the thumbnail is only displayed after downloading the whole file. Before would be a nice idea for pictures.
